Package.swift

```diff
@@ -1,11 +1,11 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.7
 // The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
 
 let package = Package(
     name: "BSImagePicker",
-    platforms: [.iOS(.v10)],
+    platforms: [.iOS(.v12)],
```

Comment:
Why is it necessary to bump iOS version to 12?

Reply:
The minimum supported version is now v12. If we don't update, the compiler throws a warning. It makes sense to get this updated to minimum supported now.